Signal/src/ViewControllers/ConversationView/Cells/OWSMessageCell.m

```diff
@@ -257,13 +257,13 @@ - (UIFont *)textMessageFont
         case 0:
             break;
         case 1:
-            return [UIFont ows_regularFontWithSize:basePointSize * 2.f];
+            return [UIFont ows_regularFontWithSize:basePointSize + 18.f];
```

Comment:
Was this intentional? It seems unrelated.

Reply:
Removed!